src/components/screens/new-questionnaire-screen.tsx

```diff
@@ -7,48 +22,90 @@ import { KeyboardReturn } from "@mui/icons-material";
 import UserRoleUtils from "src/utils/user-role-utils";
 import type { Question, QuestionOption } from "src/types/index";
 import strings from "src/localization/strings";
+import { useLambdasApi } from "src/hooks/use-api";
 
 /**
  * New Questionnaire Screen component
- * Includes a card (new-questionnaire-card.tsx) to create questions for a new questionnaire
+ * Includes a card (new-questionnaire-card.tsx) to create options for a new questionnaire
  */
 const NewQuestionnaireScreen = () => {
   const adminMode = UserRoleUtils.adminMode();
-  
+  const { questionnaireApi } = useLambdasApi();
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
   const [questionnaireTitle, setQuestionnaireTitle] = useState("");
+  const [questionnaireDescription, setQuestionnaireDescription] = useState("");
+  const [options, setOptions] = useState<Question[]>([]);
+  const [passScoreValue, setPassScoreValue] = useState(0);
 
   const handleQuestionTitleChange = (event: ChangeEvent<HTMLInputElement>) => {
```

Comment:
You could probably do something nice here where the name is passed from the textField and used in the  handler to update a single questionnaire state containing title, description and passscore- I think the options could also be updated in this way.

This would mean you have a single questionnaire state and a single handler, which updates the state according to the name attribute from the textfield component

Reply:
I was able to cast both textfield together. But Slider was a problem so I leave it as it was. Also there are no tags, so that will be task for future.

Reply:
I have added another comment to give an example of how I would do this. Hopefully you can resolve the Slider issue (I guess by slider you are referring to the pass score value state?)